packages/utils/regl-renderer/src/rendering/commands/drawMesh/index.js

```diff
@@ -71,7 +71,7 @@ const drawMesh = function (regl, params = { extras: {} }) {
 
       func: { src: 'src alpha', dst: 'one minus src alpha' }
     },
-    primitive: (context, props) => props && props.primitive ? props.primitive : 'triangles'
+    primitive: (context, props) => props && props.primitive ? props.primitive : (type === '3d' ? 'triangles' : 'lines')
```

Comment:
Hmm... I understand the change but wonder if the real correction should be made when primitives is called, I.e making sure that props.primitive is set correctly.

Reply:
that is a different thing, props.primitive is an override, and this sets the **default** for both 2D & 3D